### ISSUE TITLE
Add "groupProperties" for every task property

### ISSUE THREAD

**Issue by Ben8t** [kestra-io/kestra#6534]
### Feature description

Following #6532 , we want to add `groupProperties` to every plugin property to unclutter the display of properties in No Code editor (and potentially rigth-panel documentation)

### `1st Iteration`
Next Steps:
- [ ] Implement the groupProperties so frontend can receive the corresponding informations

1st iteration with core tasks properties and files properties:

`Core`:
> To group all common core task properties
- description
- logLevel
- retry
- timeout
- workerGroup
- allowFailure
- allowWarning
- disabled
- logToFile
- runIf
- errors (in case of flowable)
- tasks (in case of flowable and not mandatory)

`Files` (or `Scripts` - naming TBD):
> To group all properties related to script/data files (mainly for script tasks, but some edge cases appy too)
- inputsFiles
- outputFiles
- namespaceFiles

Thus, in the no code, the task panel will ends up with:

- Required task property (always displayed)
- Advanced Configuration (collapsed by default)
  - Core Task Properties
  - Files Properties

## `2nd Iteration`  

In the 2nd iteration we can move forward with:
- [ ] explore all plugins and find grouping strategies (for example: `Connection` group for all props. related to DB credentails, server auth, API token, etc.)
- [ ] add these groups to each plugins tasks

**Comment by MilosPaunovic** [comment]
**_Note:_** Assign this to me once the BE part is done, as it will have to integrate into the section I will work on.

**Comment by Ben8t** [comment]
1st iteration, we could add two group properties:

`Core Task Properties`:
> To group all common core task properties
- description
- logLevel
- retry
- timeout
- workerGroup
- allowFailure
- allowWarning
- disabled
- logToFile
- runIf
- errors (in case of flowable)
- tasks (in case of flowable and not mandatory)

`Files Property` (or `Scripts Management Properties` - naming TBD):
> To group all properties related to script/data files (mainly for script tasks, but some edge cases appy too)
- inputsFiles
- outputFiles
- namespaceFiles

Thus, task panel will ends up with:

- Required task property (always displayed)
- Advanced Configuration (collapsed by default)
  - Core Task Properties
  - Files Properties
  
2nd iteration would probably need a deeper dive as we might need to look into every plugin specific props. and find out a good grouping strategy (for example `Connection Properties` filled with all things related to DB, server, etc. credentails)

**Comment by fdelbrayelle** [comment]
This was done in [another issue](https://github.com/kestra-io/kestra-ee/issues/6712), closing it!
